src/plugins/visualizations/server/saved_objects/visualization_migrations.test.ts

```diff
@@ -30,9 +30,20 @@
 
 import { visualizationSavedObjectTypeMigrations } from './visualization_migrations';
 import { SavedObjectMigrationContext, SavedObjectMigrationFn } from 'opensearch-dashboards/server';
+import { createReplaceSerializer } from '@osd/dev-utils';
 
 const savedObjectMigrationContext = (null as unknown) as SavedObjectMigrationContext;
 
+// ToDo: Remove this logic when Node 14 support is removed
```

Comment:
needs link to issue

Reply:
Sure. This issue https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4129 contains all the 3 tests.